qmt/_version.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+# This file is part of 'miniver': https://github.com/jbweston/miniver
```

Comment:
Are these files a result of setting up miniver? How would we upgrade (if we need to)?

Reply:
Yes, this is required to get the version from the git tags.

We upgrade by just downloading the new file from that repo.